ONSdigital/eq-author-app
# Migrated Footer folder to PaginationNav

### What is the context of this PR?

> Have migrated the Footer folder and components to PaginationNav folder and renamed block in Questionnaires view so the block is correctly named and not associated as a generic footer.
>

### How to review 

> Add to the list below as appropriate, including screenshots when necessary

* Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
    * ensure it can be opened in Author;
    * then, ensure it can be viewed in Runner by pressing the **view survey** button
* Create a survey and duplicate a number of times until a second page is created
* Check the pagination controls work as expected

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Jira ticket for this task into the next stage of the process
